src/cache/redis.ts

```diff
@@ -15,7 +15,34 @@ const logger = makeLogger('RedisCache')
 export class RedisCache<T = unknown> implements Cache<T> {
   type = CacheTypes.Redis
 
-  constructor(private client: Redis) {}
+  constructor(private client: Redis) {
+    this.loadFunctions()
+  }
+
+  async loadFunctions() {
+    // Load custom function 'setResponse' with lua script to redis that will skip overwriting successful cache response if the new value is error response
+    const source =
+      '#!lua name=ea \n ' +
```

Comment:
Let's change this to a multiline string (with `) so we don't have to sum every line